service/controller/resource/masters/error.go

```diff
@@ -72,6 +72,25 @@ func IsMissingLabel(err error) bool {
 	return microerror.Cause(err) == missingLabelError
 }
 
+func IsNotFound(err error) bool {
+	if err == nil {
+		return false
+	}
+
+	c := microerror.Cause(err)
+
+	{
```

Comment:
Not sure about the block here, but it looks somehow weird. Alternative proposal:
```suggestion
	{
		c := microerror.Cause(err)
```

Reply:
agreed